src/bezier.js

```diff
@@ -148,7 +148,7 @@ function cubicBezierAtTime(t, p1x, p1y, p2x, p2y, duration) {
  *  @return {Function}
  *  @private
  */
-const getCubicBezierTransition = (x1, y1, x2, y2) => pos =>
+export const getCubicBezierTransition = (x1, y1, x2, y2) => pos =>
```

Comment:
Given that this may now potentially receive `undefined` values for parameters when called from `composeEasingObject` (in case the user provides an array with less than four elements for easing), it might be worth giving all of this function's parameters default values. It would avoid loud failures, but that seems preferable in this scenario. What do you think?

Reply:
Sounds reasonable, it is either giving it default values or silently falling back to linear easing? With that said, `[0.250, 0.250, 0.750, 0.750]` would be the default values for [linear easing](https://matthewlein.com/tools/ceaser), so either method works!

Reply:
I think using `[0.250, 0.250, 0.750, 0.750]` as default values is a great solution. :+1: